### PR TITLE
SNO-69-refactor-result-from-set_filters

### PR DESCRIPTION
**Pull request by zoldello**

### CODE REVIEW

File: src/snovault/helpers/helper.py [hunk 1]
```diff
@@ -268,18 +268,16 @@ def build_terms_filter(query_filters, field, terms):
...
     query_filters = query['post_filter']['bool']
     used_filters = {}
-    if static_items is None:
-        static_items = []
+    result_filters = []
 
     # Get query string items plus any static items, then extract all the fields
     qs_items = list(request.params.items())
-    total_items = qs_items + static_items
+    total_items = qs_items + (static_items or [])
```
Comment (caseylitton) [inline src/snovault/helpers/helper.py]: I'm not a fan of the ```+``` for list create/concatenation unless it is being used a lot in the function.  Here it's just a shortcut to create a new list from two other lists. 

I'd prefer to make it more readable with explicit steps.
```
total_items = qs_items.copy()
if static_items:
    total_items.extend(static_items)
```
    
The original code was fine too, so may be best to not change in any case.

File: src/snovault/helpers/helper.py [hunk 2]
```diff
@@ -268,18 +268,16 @@ def build_terms_filter(query_filters, field, terms):
         query_filters['must'].append(filter_condition)
 
 
-def set_filters(request, query, result, static_items=None, filter_exclusion=None):
-    """
-    Sets filters in the query
-    """
+def set_filters(request, query, static_items=None, filter_exclusion=None):
```
[the remark this answers is not in the document]
Comment (zoldello) [reply src/snovault/helpers/helper.py]: How important is having query_filters passed in a an argument to you? The method- set_filters is used outside of snovalt and already has four arguments; so adding query_filters as a parameters seems to give very little gain given what it takes to add it.

File: src/snovault/helpers/helper.py (hunk 1 above)
Comment (zoldello) [reply src/snovault/helpers/helper.py]: Agreed. Your way seems more pythonic. However, the code works and we are not trying to polish up set_filters. So it is best to leave it.

Comment (caseylitton) [reply src/snovault/helpers/helper.py]: Hey, We need to return this to it's original form.  It was an unnecessary change and we're being more strict on that now.

**Comment by zoldello** [comment]
You mentioned this: 
"Looks good. A little further clean up and maybe revert the unneeded change to the list concatenation."

Was that addressed. I took out the doc string comments.

File: .travis.yml [hunk 3]
```diff
@@ -101,7 +101,8 @@ script:
       --browser-arg-int build  "$TRAVIS_BUILD_NUMBER" \
       --browser-arg-int idleTimeout 300 \
       --browser-arg name "$TRAVIS_REPO_SLUG $TRAVIS_BRANCH $TRAVIS_COMMIT" \
-      --browser-arg browser "$BROWSER"
+      --browser-arg browser "$BROWSER" \
+      --browser-arg platform "Linux"
```
Comment (caseylitton) [inline .travis.yml]: revert this change and rebase since this is in snovault now.

Comment (zoldello) [reply .travis.yml]: I removed the two commit related to this and then rebased on master.

**Comment by caseylitton** [comment]
closed.  code is way out of date.
